project/iterative_sorting.py

```diff
@@ -23,11 +23,27 @@ def selection_sort( arr ):
 # TO-DO: implement the Insertion Sort function below
 def insertion_sort( arr ):
     for i in range(1, len(arr)):
-        cur_value = 
-        while 
+        cur_value = None
+        for  j in range(i, len(arr[::-i]), -1):
+            j > cur_value
```

Comment:
this doesn't look quite finished to me. you're doing a comparison, but no swapping. I also get errors straight away when running this.

Reply:
Just commit the latest version, forgot to update it.